docs-win/index.md

```diff
@@ -0,0 +1,297 @@
+
+2\. Click on the "Tools for Visual Studio 2019" section, and click the "Download" button next to "Build Tools for Visual Studio 2019".
+
+![Visual Studio Build Tools download](./img/01.png)
+
+3\. Execute the downloaded file to start the installation. Ensure the box for C++ build tools is checked, and the first 3 boxes on the right side under Optional. Click the Install button.
+
+Note that this downloads approximately 1.6 GB of files for installation, so it may take awhile.
+
+![Visual Studio download options](./img/02.png)
+
+### Install and verify minidsp-rs
+
+minidsp-rs is a utility, written by mrene on avsforum.com, which allows the system to communicate with the MiniDSP 2x4HD, without using the proprietary minidsp plugin. Pre-compiled binaries are available for most operating systems, and there is an available Windows executable.
+
+1\. The minidsp-rs software is in a .tar.gz compressed format, which Windows does not understand natively. Unless a more robust decompression utility is already installed (in which case, skip this step), download and install 7-Zip ([www.7-zip.org](https://www.7-zip.org)), which is a free decompression utility that supports many more formats than the native Windows tools.
+
+2\. Open a browser and go to [https://github.com/mrene/minidsp-rs/releases](https://github.com/mrene/minidsp-rs/releases). Download file minidsp.x86_64-pc-windows-msvc.tar.gz
+
+![minidsp-rs site](./img/03.png)
+
+3\. Open the .tar.gz file with 7-Zip. The easiest method is to right click on the file, go to 7-Zip, and select Open archive.
+
+![Example decompression of the file](./img/04.png)
+
+4\. A .tar.gz file is actually 2 containers. The first one (the .gz) is now open in 7-Zip.
+
+![Example decompression of the file](./img/05.png)
+
+5\. Double-click on the file in the 7-Zip window, to open the .tar file.
+
+![Example decompression of the tar file](./img/06.png)
+
+6\. Highlight the minidsp.exe file, and click the Extract icon. Select a location for the file (such as the user's home directory) and make note of it as it will be needed for the ezbeq configuration.
+
+![Example extraction](./img/07.png)
+
+7\. To confirm basic functionality, open a Command Prompt.
+
+![Open a Command Prompt](./img/08.png)
+
+8\. In the Command Prompt window, type: `minidsp`
+
+![Example of minidsp output](./img/09.png)
+
+5\. If the MiniDSP 2x4HD is connected properly, similar information to the screenshot above will be displayed. Note the &quot;preset: 1&quot; in the image. That indicates that the MiniDSP 2x4HD is set to Config slot 2. The minidsp-rs application starts at 0 for the config presets, so 0 = Config slot 1, and so on.
+
+6\. If the MiniDSP 2x4HD is not detected, or is not connected, the following error will appear.
+![Example of minidsp error message](./img/10.png)
+
+### Install Chocolatey
+
+Chocolatey is a package manager for Windows, which will be used to install Python.
```

Comment:
fwiw you can use choco for 7z too - https://chocolatey.org/packages/7zip.install

Reply:
I'll test that out. Makes sense.